src/platform/forms-system/src/js/widgets/SelectWidget.jsx

```diff
@@ -34,7 +34,9 @@ function SelectWidget({
       id={id}
       name={id}
       multiple={multiple}
-      className={options.widgetClassNames}
+      className={`${options.widgetClassNames} ${
```

Comment:
enabled vs disabled with this change:
![image](https://user-images.githubusercontent.com/20728956/73681734-d8297380-4673-11ea-8f5e-f82e7c85d526.png)
This new look is already being done on the existing profile address update form

Reply:
For what it's worth, we've intentionally shied away from the "disabled" pattern in the past. :man_shrugging: